### DISCUSSION TITLE
Vercel Preview works on branch but not after merge (date format problem)

### DISCUSSION THREAD

**Discussion by AntjeWalter** [neuefische/questions#444] [Web] [answered]
Hello everyone,
I worked with date-fns for formatting my birthdays and it worked perfectly fine on the localhost and also in the vercel preview of the branch. However, now after merging my PR, the vercel preview doesn't work anymore and it says the following:
![Bildschirmfoto 2023-01-09 um 14 37 50](https://user-images.githubusercontent.com/115352305/211320753-d1d5f65d-63f2-4fd1-8c62-15c6db85b202.png)

I really can't understand how this is possible after both the localhost and the branch-preview worked. I also can only check if something has changed if I merge every change I do, which is not really the best way to troubleshoot... Maybe somebody has an idea?

The problem is in line 65 in here, I guess: https://github.com/AntjeWalter/capstone-project/blob/main/components/BirthdayListPage/BirthdayList.js

**Comment by pixelass** [comment]
Looks like the format string is not correct `"dd'.'MM'.'yyyy"`. Try removing the extra `'`. `"dd.MM.yyyy"`

**Reply by pixelass** [reply]
It might also be that the dynamic value `birthday` is not using a valid date (that can be parsed by the library "date-fns");

**Reply by AntjeWalter** [reply]
Unfortunately your first suggestion didn't work.

But while trying your second, I just realized that Vercel has different domains. [This one](https://capstone-project-gamma-eight.vercel.app/) is connected with my Github repository on the right and is the one that is not working. However [this one](https://capstone-project-il1sngijc-antjewalter.vercel.app/) and [this one ](https://capstone-project-antjewalter.vercel.app/) are also listed on Vercel as domains and those do work for me and don't show the mistake.

I don't really understand why there are different domains and why only one of them is not working. Do you have any idea why that is?

**Reply by pixelass** [reply]
Vercel has one main domain and then several additional (one for each deployment, one for each branch, on extra for the user/repo). It is very strange that one environment behaves different than the other since they are just servers and your app is executed in a browser. I am pretty sure that something else is causing this issue. 


I tested your domain and could not reproduce the issue: 

**My best guess is that you have broken data in your `localStorage` for that domain. Can you delete the localStorage and see if the issue persists?**

![Screenshot 2023-01-09 at 17 51 15](https://user-images.githubusercontent.com/1148334/211363059-11dd3455-6d41-4821-a716-caa2ce348fb7.png)

![Screenshot 2023-01-09 at 17 51 40](https://user-images.githubusercontent.com/1148334/211362994-b40b470f-9789-4c24-84a0-73ef96c065a4.png)

**Reply by AntjeWalter** [reply]
You were right, I cleared the local storage and now it works again! Thank you so much! Saved my evening! ;)

**Comment by AntjeWalter** [comment, marked as answer]
For the correct documentation in this discussion board: I cleared my local storage and now it works again. Seems like there was some broken data in there.
